### PR TITLE
part of 92, convert form feilds to stateless widgets

### PR DESCRIPTION
**Pull request by tamari-gray**
attempt at refactoring #92 

sorry if i overstepped into your work, once i got stuck in it was really fun

### CODE REVIEW

File: test/create_game_section/new_game1/NewGameViewModel1_unit_test.dart [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Import the test package and Counter class
+import 'package:niira/services/game_service.dart';
```
Comment (tamari-gray) [inline test/create_game_section/new_game1/NewGameViewModel1_unit_test.dart:2]: very unneccessary unit test, but it helped me understand how we are using the newgameViewmodel in gameservice. 
then thought its not bad as a smoke test

**Comment by nickmeinhold** [comment]
Glad you enjoyed it!  

I was trying to remove the weirdness of the potential assign to null - what you originally asked about in my PR.  The mention of moving to stateless was a bit of a red herring because the goal was to get rid of the weirdness but I thought changing to stateless was part of the way to get there.  

So while this does get us to stateless (nice job!) it doesn't remove the weirdness.  Let's have a chat about how to move forward when you're free. I'm free all day.

**Comment by tamari-gray** [comment]
ahhhh cool! yeah im super keen!
will just have some lunch and give you a buzz :)

**Comment by nickmeinhold** [comment]
Great!  I posted in the other thread about this but the weirdness wasn't weirdness at all, my bad!  

I'm going to merge this then, caus it's great - awesome job!

**Pull request merged**